src/js/components/dropdownButton.js

```diff
@@ -1,34 +1,19 @@
-document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Without this event listener, does calling code need to manually activate this behavior now? I tried to look at how `behavior()` in USWDS was implemented and didn't see anything that listened for DOM ready

Reply:
> Without this event listener, does calling code need to manually activate this behavior now? I tried to look at how `behavior()` in USWDS was implemented and didn't see anything that listened for DOM ready

It's a bit indirect, but no, a consumer of the existing `dist/assets/js/main.js` won't need to change anything to initialize the dropdown button. It'll still happen on `DOMContentLoaded` automatically.

We handle this by iterating through `uswds.components` on `domready` to initialize each component, which has effectively the same result as what existed previously:

https://github.com/18F/identity-style-guide/blob/d900161552f4b2990a07f5f1009c1a776804ec36/src/js/main.js#L21-L28

With #165, a consumer would need to initialize it themselves if they use the component through the new package entrypoint, but that's intended, and also [documented](https://federalist-2f194a10-945e-4413-be01-46ca6dae5358.app.cloud.gov/preview/18f/identity-style-guide/aduth-lg-3716-package-entry/usage/#use-as-a-javascript-package).

Reply:
Ah perfect, thanks for clarifying